COMMITTERS.md

```diff
@@ -87,7 +85,7 @@ Branch name should be:
 * Successfully passed review.
 * Rebase in case of existing conflicts.
 * Contributor can squash commits in case of adding same code in different commits.

```

Comment:
This document as a whole looks like it is specifically for use by Luxoft developers, I have removed it in #1800. If it is still needed by Luxoft, it might be better if it were moved to another repository (maybe `LuxoftSDL` or `sdl_requirements`) as it is a bit confusing to keep it here considering CONTRIBUTING.md already exists.

In any case, since this is deleted in the new release, please remove any changes to this file in this PR.

Reply:
@jacobkeeler 
Done with 096051f